src/time/time.cr

```diff
@@ -1,3 +1,50 @@
+# Time library allows you to inspect, analyze, and calculate time. Here are some examples:
```

Comment:
"The Time library", "format time" too ;) and maybe we should mention time spans (`Time::Span`) too?